client/network/src/config.rs

```diff
@@ -445,6 +461,67 @@ where
 	fs::OpenOptions::new().write(true).create_new(true).open(path)
 }
 
+/// The configuration of a node's WebRTC certificate, describing how it is obtained.
+#[derive(Clone, Debug)]
+pub enum WebRTCConfig {
+	/// A WebRTC certificate.
+	File(Option<PathBuf>),
+}
+
+impl Default for WebRTCConfig {
+	fn default() -> WebRTCConfig {
+		Self::File(None)
+	}
+}
```

Comment:
I'm not sure that we should provide a default. You are very much supposed to load a certificate from disk (at least until the deterministic certificate system lands).

Reply:
I've added `Ephemeral` option for tests. Let me know if you think we should use `File` option in all tests too.